src/engine/renderer/glsl_source/computeLight_fp.glsl

```diff
@@ -239,8 +239,8 @@ void computeDynamicLights( vec3 P, vec3 normal, vec3 viewDir, vec4 diffuse, vec4
 	// NOT the amount of lights that can actually be put into each layer of the lighttile texture
 	uint globalLightsPerLayer = ( u_numLights + numLayers - 1 ) / numLayers;
 
-	uint lightCount = 0;
 	for( uint layer = 0; layer < numLayers; layer++ ) {
+		uint lightCount = 0;
 		idxs_t idxs = fetchIdxs( tileScale * vec3( tile, float( layer ) + 0.5 ), u_LightTiles );
 
 		uint lightOffset = layer * globalLightsPerLayer;
```

Comment:
Why don't we make the loop condition below just `i < 16`? It will break on idx being 0 anyway.

Reply:
I suppose that's better, yeah.

Reply:
Done now.